docs/guides/1.4.0-upgrade-guide.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: "couchbase-capella"
+page_title: "Couchbase Capella Provider 1.4.0: Upgrade and Information Guide"
+sidebar_current: "docs-couchbase-capella-guides-140-upgrade-guide"
+description: |-
+Couchbase Capella Provider 1.4.0: Upgrade and Information Guide
+---
+
+
+# Couchbase Capella Provider 1.4.0: Upgrade and Information Guide
+
+New features in 1.4.0
+
+
+## New Features
```

Comment:
Should this be after the heading?

```suggestion

## New Features

New features in 1.4.0:

```

Reply:
+1 to @simon-dew  suggestion

Reply:
@simon-dew We can update it but in the past release we have done it like this only

New features in <Version Number> and in next line we have added "New Features" as heading text.

Attaching a screenshot of last release for reference. 
<img width="873" alt="Screenshot 2024-11-25 at 1 00 10 PM" src="https://github.com/user-attachments/assets/1fd63924-2aad-4032-b1bb-73f93ee798fb">

Reply:
Fair enough! I'm happy to keep it as it is for consistency